src/components/NavBar/NavigationBar.cy.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import NavigationBar from "@/components/NavBar/NavigationBar";
+import { AppRouterContext, AppRouterInstance } from "next/dist/shared/lib/app-router-context";
+
+describe("<NavigationBar />", () => {
+    const router: AppRouterInstance = {
+        push: () => {},
+        back: () => {},
+        forward: () => {},
+        replace: () => {},
+        prefetch: () => {},
+        refresh: () => {},
+    };
+
+    const RouterWrappedNavBar = (
+        <AppRouterContext.Provider value={router}>
+            <NavigationBar />
+        </AppRouterContext.Provider>
+    );
+
+    it("Renders on desktop", () => {
+        cy.viewport("macbook-13");
+        cy.mount(RouterWrappedNavBar);
+    });
+
+    it("Renders on mobile", () => {
+        cy.viewport("iphone-se2");
+        cy.mount(RouterWrappedNavBar);
+    });
+
+    it("Links exist on desktop", () => {
+        cy.viewport("macbook-13");
+        cy.mount(RouterWrappedNavBar);
+
+        cy.get("a[href='/clients']").contains("Clients").should("exist");
+        cy.get("a[href='/lists']").contains("Lists").should("exist");
+        cy.get("a[href='/calendar']").contains("Calendar").should("exist");
+    });
+
+    it("Links exist on mobile", () => {
+        cy.viewport("iphone-se2");
```

Comment:
preferred when it was a viewport defined by pixels - but happy to take whatever dimensions this is (as long as it's in pixels!)

Reply:
Have reverted